File: test-explorer/v2x_net452_FSharp/BasicTests.fs
namespace FSharp

open Xunit

type BasicTests() =
    [<Fact>]
    member x.Passing() =
        Assert.True true

    [<Fact>]
    member x.Failing() =
        Assert.True false

    [<Theory>]
    [<InlineData("hello from v2x_net452_FSharp")>]
    [<InlineData("")>]
    member x.Theory(value:string) =
        Assert.NotEmpty value

File: msbuild/v3x_net60/BasicTests.cs
using Xunit;

public class BasicTests
{
    [Fact]
    public void Passing()
    {
        Assert.True(true);
    }

    [Fact]
    public void Failing()
    {
        Assert.True(false);
    }

    [Theory]
    [InlineData(2112)]
    [InlineData("hello from v3x_net60")]
    [InlineData(new object?[] { null })]
    public void Theory(object? value)
    {
        Assert.NotNull(value);
    }
}
